code/modules/defenses/sentry.dm

```diff
@@ -22,10 +23,29 @@
 	var/omni_directional = FALSE
 	var/sentry_range = SENTRY_RANGE
 
+	has_camera = TRUE
+	camera_range = 7
+
 	var/damage_mult = 1
 	var/accuracy_mult = 1
+	var/burst = 1
 	handheld_type = /obj/item/defenses/handheld/sentry
 
+	var/enaged_timer = null
+	var/low_ammo_timer = null
+	var/sent_empty_ammo = FALSE
+
+	// action list is configurable for all subtypes, this is just an example
+	choice_categories = list(
+		"RATE OF FIRE" = list("SINGLE", "BURST", "FULL-AUTO"),
+		"IFF STATUS" = list("USCM", "WY", "HUMAN"),
```

Comment:
use defines for "single" etc instead of strings

Reply:
I've moved all of these strings to defines